src/content/blog/docker-providers.md

```diff
@@ -0,0 +1,58 @@
+
+macOS Docker installation for each of these is [in the docs](https://ddev.readthedocs.io/en/latest/users/install/docker-installation/#macos); the Docker providers and their differences are shown here:
+
+| Provider                                                          | Advantages                                      | Disadvantages                                       | Open Source? | Free? |
+|-------------------------------------------------------------------|-------------------------------------------------|-----------------------------------------------------| --- | --- |
+| [OrbStack](https://orbstack.dev)                                  | Lightweight, great performance, well maintained |                                                     | No | No |
+| [Colima](https://github.com/abiosoft/colima)                      | Many, many permutations of configuration        | There have been a variety of problems with upgrades | Yes | Yes |
+| [Docker Desktop](https://www.docker.com/products/docker-desktop/) | Well-known                                      | Often buggy                                         | No | No |
+| [Rancher Desktop](https://rancherdesktop.io/)                     |                                                 | New to DDEV                 | Yes | Yes |
+
+## What is a Docker Provider?
+
+DDEV depends on a Docker Provider, but what is it?
+
+Docker is the reason that DDEV can behave the same on every operating system, including Linux, WSL2, traditional Windows, and macOS (both Apple Silicon and Intel).
+
+All of the Docker Providers on every platform (except Linux) are actually wrappers on the open-source [Docker/Moby project](https://github.com/moby/moby), which is supported by [Docker, Inc](https://docker.com). Thank you! On Linux, the Docker setup runs natively, without any wrapper, which is why DDEV is so performant on Linux.
+
+## How can DDEV support all these providers?
+
+The idea of Docker is that everything happens inside a container, which you can think of as a little, separate computer. So the Docker images and containers used by DDEV are the same on every operating system, except that we build separate images for each [processor architecture](blog/arm64-apple-silicon-m1-ddev-local-what-does-it-all-mean/) (ARM64 - Apple Silicon, and AMD64 - Intel). So mostly all the work is done inside the container, and the containers/images behave the same. And DDEV’s main code is written in Golang, which is compiled natively into a single binary on every platform. Of course that sounds simple, but it’s not all that simple. Building all those images has quite a history, and of course Docker providers that ought to behave the same everywhere don’t actually, so there are workarounds in many places.
```

Comment:
Minor suggestions for flow, clarity, the concept of multiple containers, and reassurance that DDEV is the one managing workarounds so you don’t have to:

```suggestion
The idea of Docker is that everything happens inside a container, which you can think of as a little, separate computer. You can have as many of them as you want, and they can be created and destroyed easily.

DDEV’s Docker images and containers are the same on every operating system, except that we build separate images for each [processor architecture](blog/arm64-apple-silicon-m1-ddev-local-what-does-it-all-mean/) (ARM64 = Apple Silicon, and AMD64 = Intel). So mostly all the work is done inside the container, and the containers/images behave the same. And DDEV’s main code is written in Golang, which is compiled natively into a single binary on every platform. Of course that sounds simple, but it’s not all that simple. Building all those images has quite a history, and of course Docker providers that ought to behave the same everywhere don’t actually, so there are workarounds in many places. (But you don’t have to worry about them _using_ DDEV!)
```

Is it too reductive to equate ARM64 and Apple Silicon? A Raspberry Pi isn’t Apple Silicon but it is ARM64, right?

Reply:
It might be. We test on linux arm64, of course, and a RPI is the main gateway to the buildkite test runners. But for most people, Apple Silicon == ARM64.